src/Graphics/sfml-graphics.csproj

```diff
@@ -26,7 +26,7 @@
 		<IsOSX>False</IsOSX>
 		<IsAndroid>False</IsAndroid>
 		<IsLinux>False</IsLinux>
-		<Version>2.4.2</Version>
+		<Version>2.5.1</Version>
```

Comment:
As this will be the first release it should be SFML.Net 2.5.0, that it uses SFML 2.5.1 underneath is irrelevant, meaning you could just as well use SFML 2.5.0 and it won't make a difference.